_pages/algorithm-development/source-control-management.md

```diff
@@ -454,24 +454,22 @@ To host your algorithm on Bitbucket Server, simply select the Bitbucket Server i
 
 Once you've selected your Bitbucket Server instance, you will be able to host algorithm source code in Bitbucket Server.
 
-You can customize two aspects of the Bitbucket Server repositories that are created for your algorithms: the repository's owner, and its name.
+You can customize two aspects of any Bitbucket Server repository that's created for an algorithm: the repository's owner, and its name.
 
-By default, all Bitbucket Server respositories are the project configured by your administrator. These proojects are listed in the "Repository Owner" dropdown:
+By default, all Bitbucket Server repositories are the project configured by your administrator. These projects are listed in the "Repository Owner" dropdown:
```

Comment:
This first sentence still doesn't make sense to me; I think it's missing a word. Is it supposed to say "....are *in* the project configured by your admin"?

Reply:
```suggestion
By default, all Bitbucket Server repositories are the project configured by your administrator. These projects are listed in the "Repository Owner" drop-down:
```

Reply:
yeah, that doesn't sound right. I need to go through the flow and think about better wording.